webclient/app/src/app/commons/alert/ConfirmationDialog.jsx

```diff
@@ -1,32 +1,60 @@
 import * as React from 'react';
+import {useState} from 'react';
 import Button from '@mui/material/Button';
 import Dialog from '@mui/material/Dialog';
 import DialogActions from '@mui/material/DialogActions';
-import Alert from '@mui/material/Alert';
-import AlertTitle from '@mui/material/AlertTitle';
-import { useTranslation } from 'react-i18next';
+import {useTranslation} from 'react-i18next';
+import {DialogContent, DialogContentText, DialogTitle} from "@mui/material";
+import PropTypes from "prop-types";
+import {LoadingButton} from "@mui/lab";
 
 export default function ConfirmationDialog(props) {
-    const { content, open, onClose, onSubmit, title, message } = props;
-    const { t } = useTranslation();
+    const {open, onClose, onSubmit, title, message, confirmTitle} = props;
+    const {t} = useTranslation();
+    const [isProcessing, setIsProcessing] = useState(false);
+
+    function confirmAction() {
+        setIsProcessing(true);
+        onSubmit()
+            .then(() => {
+                onClose();
+                setIsProcessing(false);
+            })
+            .catch(() => {
+                setIsProcessing(false)
+            })
+    }
 
     return (
         <Dialog
             open={open}
             onClose={onClose}
-            aria-labelledby="alert-dialog-title"
```

Comment:
can we please use this kind of text:

![image](https://user-images.githubusercontent.com/10485792/159028394-51f7b96b-e61c-42f8-85b1-1e4fc5b0785e.png)

Reply:
This styling is not very Material UI compliant and in my opinion quite confusing.
If you insist, I can revert that.
https://material.io/components/dialogs#alert-dialog